### DISCUSSION TITLE
Possible to show/hide columns in tree view?

### DISCUSSION THREAD

**Discussion by Tuuuukka** [cmus/cmus#1324] [Troubleshooting] [answered]
So I'd like to show e.g. a "playcount" or a "last played at" column in the tree view, like this:

![image](https://github.com/cmus/cmus/assets/10988493/5d65f885-9914-4759-aa4f-86b454c52d55)

Is it possible to alter the columns that are displayed? I couldn't find anything on this, but I might've just missed it.

**Comment by Lagrang3** [comment]
There is a play count. The format variable `%{X}` or `%{play_count}` shows it. For example this is the value of my `format_playlist`:
```
%-30{album} %20%a %3n. %t%= %y %d %{?X!=0?%3X ?    }
```

**Reply by Tuuuukka** [reply]
Hmmmm, does that affect the `1` view? Or just the playlist (`3`) view? Although I don't know why, but I can't get it working even on the playlist view 🤔

**Reply by Lagrang3** [reply, marked as answer]
Testing by changing the format string, I could deduce that `format_playlist` and `format_playlist_va` would affect views 2, 3 and 4,
for view 1, you'll need to change `format_trackwin` and `format_trackwin_va`. And yes the counter displays.

I suggest you just don't copy paste my format string, I am not 100% aware of the syntax. Try to construct your own according to your needs.

**Reply by Tuuuukka** [reply]
Ah, it was the `format_trackwin`, now it works as I wanted, thank you!
